corehq/apps/app_manager/views/phone.py

```diff
@@ -0,0 +1,70 @@
+from django.http import HttpResponse
+from eulxml.xmlmap import XmlObject
+
+from django.conf import settings
+
+from corehq.apps.domain.auth import basicauth
+from corehq.apps.domain.decorators import check_lockout
+from corehq.apps.users.models import CouchUser
+from corehq.apps.app_manager.dbaccessors import get_built_app_ids, get_app
+from corehq.apps.app_manager.suite_xml.xml_models import (
+    StringField,
+    IntegerField,
+    NodeListField,
+)
+
+
+class App(XmlObject):
+    ROOT_NAME = 'app'
+
+    domain = StringField('@domain')
+    name = StringField('@name')
+    environment = StringField('@environment')
```

Comment:
Mobile polls both prod and india for your available apps, so this was intended to show where the app was coming from.